README.md

```diff
@@ -14,39 +14,11 @@ REL utilizes *English* Wikipedia as a knowledge base and can be used for the fol
 
 Documentation: <https://rel.readthedocs.io>
 
-To install:
-
-```console
-pip install radboud-el
-```
-
 The source code is available from [Github](https://github.com/duqtools/duqtools).
```

Comment:
This cannot be right!

Reply:
This is an old commit, you can ignore it.